### PR TITLE
Make artifacts executable

### PR DESCRIPTION
**Pull request by CodeSandwich**
Fixes https://github.com/radicle-dev/radicle-registry/issues/379 by tarballing the artifacts

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -106,8 +106,8 @@ Make a release
     - Expand the `Test ci/run` section
     - Open the `Artifacts` tab
     - Download the following binaries:
-        - `radicle-registry-cli`
-        - `radicle-registry-node`
+        - `radicle-registry-cli.tar.gz`
+        - `radicle-registry-node.tar.gz`
```
Comment (geigerzaehler) [inline DEVELOPING.md:110]: Does it make sense to have two tarballs or should we combine them into one?

Comment (NunoAlexandre) [reply DEVELOPING.md:110]: It makes sense to have two, and every time I saw binaries provided this way a `zip` counterpart is also made available.

Comment (CodeSandwich) [reply DEVELOPING.md:110]: I was thinking about it and to be honest I don't know. It probably doesn't make sense to download only one. I'll merge them.

Comment (NunoAlexandre) [reply DEVELOPING.md:110]: No. we simply shouldn't force users to download two things if they only need one.

Comment (CodeSandwich) [reply DEVELOPING.md:110]: Sorry, I didn't see your comment, GitHub is sometimes a bit asynchronouse. Either way is good for me.

Comment (NunoAlexandre) [reply DEVELOPING.md:110]: We really shouldn't merge it, especially given that the node binary is like 10x bigger than the cli binary. That is a real UX issue. Running a tar command vs chmod +x command seems way more subjective to me.

Comment (geigerzaehler) [reply DEVELOPING.md:110]: Either way is fine with me.

Comment (CodeSandwich) [reply DEVELOPING.md:110]: For the sake of conversation: it's 8 and 38.5 MB or 2.5 and 14 MB if compressed.

**Pull request merged**